_conferences/devfest-istanbul-2019.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "GDG DevFest"
+website: https://devfest.istanbul/
+location: Istanbul, Turkey
+
+date_start: 2019-11-24
+date_end:   2019-11-24
+
+cfp_start: 2019-09-02
+cfp_end:   2019-11-24
```

Comment:
Can you change it to 2019-19-24. It is the actual CFP end date.

Reply:
CFP end date is updated.